documentation/modules/vmware-privileges.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+[id="vmware-privileges_{context}"]
+= VMware privileges
+
+The following VMware privileges are required to migrate virtual machines to {virt} with the {project-first}.
+
+[cols="2", options="header"]
+.VMware privileges
+|===
+|Privilege |Description
+|`Virtual machine.Interaction` privileges:|
+|`Virtual machine.Interaction.Power Off`   |Allows powering off a powered-on virtual machine. This operation powers down the guest operating system.
```

Comment:
It sounds like this is to gracefully shut down a VM. Is there an option to do a hard power-off for a frozen VM? (Like holding down the power button on your laptop if it freezes).

Reply:
@stoobie Not that I know of